src/egon/data/datasets/gas_grid.py

```diff
@@ -260,9 +353,11 @@ def insert_gas_pipeline_list(gas_nodes_list):
     # Select the links having at least one bus in Germany
     gas_pipelines_list = gas_pipelines_list[
         gas_pipelines_list["country_code"].str.contains("DE")
-    ]  # A remplacer evtmt par un test sur le NUTS0 ?
+    ]
 
     # Remove links disconnected of the rest of the grid, until the SciGRID_gas data has been corrected.
+    # TODO: automatic test for disconnected links
+    # TODO: remove link test if length = 0

```

Comment:
Are these ToDos still pending?

Reply:
Yes, they are. Here is the new issue: https://github.com/openego/eGon-data/issues/590